nifi-nar-bundles/nifi-jwt-bundle/nifi-jwt-processors/src/test/resources/2048key1.key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Why are there multiple keys for each key-length?

Reply:
When first testing the processor I wanted to be sure that it handled multiple public keys in the directory correctly.